src/collection/pages/main/main.tsx

```diff
@@ -272,19 +267,11 @@ const Main: React.FC = () => {
 
     if (!latestCollectionItem) return;
 
-    if (data.stateQuery.agent!.monsterCollectionLevel < latestCollectionItem.tier) {
-      const collectionResult = await collect({
-        variables: { level: latestCollectionItem.tier },
-      });
-      return collectionResult.data!.action!.monsterCollect as string;
-    } else if (data.stateQuery.agent!.monsterCollectionLevel > latestCollectionItem.tier) {
-      const collectionResult = await cancelCollection({
-        variables: { level: latestCollectionItem.tier },
-      });
-      return collectionResult.data!.action!.cancelMonsterCollect as string;
-    } else {
-      return;
-    }
+    const collectionResult = await collect({
+      variables: { level: latestCollectionItem.tier },
+    });
```

Comment:
IMO We need to check the level from GraphQL and `latestCollectionItem` is not the same. If levels are the same, returned null is the previous spec.

Reply:
I'll add a check on the next PR.